masakari-controller/controller/masakari_config.py

```diff
@@ -179,9 +267,15 @@ def _log_setup(self):
 
         level = self.conf_log.get('log_level')
 
+        logging_context_format_string = '%(asctime)s.%(msecs)03d %(process)d %(levelname)s ' \
+            + '%(name)s [%(thread_name)s %(record_identifier)s] %(message)s'
         logging.set_defaults(
+            logging_context_format_string=logging_context_format_string,
             default_log_levels=logging.get_default_log_levels() +
             ['controller=' + level])
+        RequestContext(thread_name=str(
```

Comment:
Question. Where the context instance is assigned? _request_store variable holds the thread name and output the name in log, doesn't it.